docs/explanations/principles.md

```diff
@@ -0,0 +1,106 @@
+
+
+# Gutenberg Principles
+
+This document has been written so that those contributing to the Gutenberg project can have some understanding of what guides decisions around what is included in the project (or not).
+
+An important thing to keep at top of mind when reviewing these principles is that they are not isolated from each other. There is a cohesiveness and complementary nature between them that results in a synergistic impact and direction.
+
+When applied, sometimes these principles build on each other much like the idea of constructing a house with a foundation, walls and roof. However, other times the application of the principles results in something that is more similar to a tapestry where threads of each principle are embedded together and evident in the finished feature or design of the project.
+
+
+## Aim for delightful, consistent, accessible user interfaces for empowering content creation and expression.
```

Comment:
I adhere to "empowering content creation and expression" but I'm not sure the other concepts capture the right concepts. 🤔

Reply:
What about something like:

> The user experience is a canvas that empowers content creation and expression through intuitive and exploration friendly interfaces

Reply:
See d6db504.